src/openrct2-ui/windows/TileInspector.cpp

```diff
@@ -2094,6 +2093,7 @@ static void window_tile_inspector_paint(rct_window* w, rct_drawpixelinfo* dpi)
                 int16_t wallType = tileElement->AsWall()->GetEntryIndex();
                 DrawTextBasic(dpi, screenCoords, STR_TILE_INSPECTOR_WALL_TYPE, &wallType, { COLOUR_WHITE });
 
+                // Banner info
```

Comment:
Why did you add this comment stating the bleeding obvious?

Reply:
When i rebased i noticed that line has disappeared in the previous pr so readded it

Reply:
Yes, and? Why would you readd a comment that states the bleeding obvious?

Reply:
Just remove it?

Reply:
If you look at the rest of the file every single section of the drawing on the tile inspector has a little comment to say what that section is about. If i remove it again it will look a bit odd that this one section isn't commented. The tile inspector has a lot of special drawing for all of the different situations it has to handle so i feel these excessive comments in this case makes sense.